pages/linux/wodim.md

```diff
@@ -0,0 +1,24 @@
+# wodim
+
+> Command for record/burning data using a optical drive to CD or DVD.
+> Some usage of wodim can cause destructive actions, such as erasing all the data on a disc.
+
+- Display optical drives avaliable to wodim:
+
+`wodim --devices`
+
+- Display help switches for use with wodim:
+
+`cdrecord`
+
+- Record a pure CD-DA audio:
+
+`wodim dev=/dev/{{optical_drive}} -audio {{track*.cdaudio}}`
+
+- Burn a file to disc in optical drive where the disc will be ejected after recording (this is needed for some devices):
```

Comment:
I think we might be able to word this a bit better. I'm thinking something along the lines of this:

```
Burn a file to a disc, ejecting the disc once done (some recorders require this):
```